fdbsyncd/fdbsyncd.cpp

```diff
@@ -67,14 +92,28 @@ int main(int argc, char **argv)
             {
                 s.select(&temps);
 
-                if(temps == (Selectable *)sync.getFdbStateTable())
+                if (temps == (Selectable *)sync.getFdbStateTable())
                 {
                     sync.processStateFdb();
                 }
                 else if (temps == (Selectable *)sync.getCfgEvpnNvoTable())
                 {
                     sync.processCfgEvpnNvo();
                 }
+                else if (temps == &replayCheckTimer)
+                {
+                    if (sync.getFdbStateTable()->empty() && sync.getCfgEvpnNvoTable()->empty())
+                    {
+                        sync.getRestartAssist()->appDataReplayed();
+                        SWSS_LOG_NOTICE("FDB Replay Complete");

```

Comment:
removeSelectable for replayCheckTimer?

Reply:
Also, since the replaychecktimer and reconciliation timer are in parallel,  what is the consequence if reconciliation timer is up, but we haven't replayed?   If replay is must, but not yet done after reconciliation timer,  we should log the error and raise.

Reply:
will remove replayCheckTimer and start recontillation timer after replay is done.